pkgs/tools/misc/dt-shell-color-scripts/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, bash
+, makeWrapper
+}:
+  stdenv.mkDerivation {
+    pname = "dt-shell-color-scripts";
+    version = "1.0.0";
+    src = fetchFromGitLab {
+      # https://gitlab.com/dwt1/shell-color-scripts
+      owner = "dwt1";
+      repo = "shell-color-scripts";
+      sha256 = "sha256-bd3NBf99rCiADUKQb6fzCBDaKBmYaZHcO4qokm/39do=";
+      # have to use revesion because this package has no version tags so I am pinning the latest commit
+      rev = "fcd013ea2e1ff80e01adbcea9d0eaf9c73db94c0";
+    };
+    meta = with lib; {
+      description = "A collection of shell color scripts collected by dt (Derek Taylor)";
+      homepage = "https://gitlab.com/dwt1/shell-color-scripts";
+      license = with licenses; [ mit ];
+      maintainers = with maintainers; [ drzoidberg ];
+    };
+    buildInputs = [ bash ];
+    nativeBuildInputs = [ makeWrapper ];
+    installPhase = ''
+      # Adjust shebangs in all files to comform to nix stadards
+      find . -type f -exec sed -i "1s/.*/#!\/usr\/bin\/env bash/" {} +
```

Comment:
This is the most unclean `patchShebangs` I have ever seen.

Reply:
did not know that patchShebangs existed